### PR TITLE
opab/fn axiom 10-12 saves

### PR DESCRIPTION
**Pull request by icecream17**
Axiom diff for commit 1: https://github.com/icecream17/Stuff/blob/main/math/~ax2.diff
Axiom diff for commit 2: https://github.com/icecream17/Stuff/blob/main/math/~ax3.diff

edit: note: elopabw comes from understanding KP's edit of ssrel

<details><summary>axiom graph</summary>

image

interesting, there's visible compression

</details>

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -52394,23 +52438,34 @@ necessary if all involved classes exist as sets (i.e. are not proper
...
+    $d R x $.  $d R y $.
     $( Membership in an ordered-pair class abstraction defined by a binary
-       relation.  (Contributed by AV, 16-Feb-2021.) $)
+       relation.  (Contributed by AV, 16-Feb-2021.)  (Proof shortened by SN,
+       11-Dec-2024.) $)
     elopabr $p |- ( A e. { <. x , y >. | x R y } -> A e. R ) $=
-      ( cv wbr copab wcel cop wceq wa wex elopab df-br biimpi eleq1 syl5ibr imp
-      exlimivv sylbi ) CAEZBEZDFZABGHCUAUBIZJZUCKZBLALCDHZUCABCMUFUGABUEUCUGUCU
-      GUEUDDHZUCUHUAUBDNOCUDDPQRST $.
+      ( cv wbr copab opabss sseli ) AEBEDFABGDCABDHI $.
```
Comment (wlammen) [inline set.mm:52468]: This not only shortens the proof, but reduces dependencies on axioms as well, and removes disjointness conditions.  Perhaps use a revision tag instead.

File: set.mm [hunk 2]
```diff
@@ -52394,23 +52438,34 @@ necessary if all involved classes exist as sets (i.e. are not proper
...
-       (Contributed by Stefan O'Rear, 20-Feb-2015.) $)
+       (Contributed by Stefan O'Rear, 20-Feb-2015.)  Avoid ~ ax-sep ,
+       ~ ax-nul , ~ ax-pr .  (Revised by SN, 11-Nov-2024.) $)
     iunopab $p |- U_ z e. A { <. x , y >. | ph } =
         { <. x , y >. | E. z e. A ph } $=
+      ( vw cv copab wcel wrex cab cop wceq wa wex ciun wb rexcom4 exbii bitri
+      cvv elopabw elv rexbii r19.42v abbii df-iun df-opab 3eqtr4i ) FGZABCHZIZD
+      EJZFKUJBGCGLMZADEJZNZCOZBOZFKDEUKPUOBCHUMURFUMUNANZCOZBOZDEJZURULVADEULVA
+      QFABCUJUAUBUCUDVBUTDEJZBOURUTDBERVCUQBVCUSDEJZCOUQUSDCERVDUPCUNADEUESTSTT
+      UFDFEUKUGUOBCFUHUI $.
```
Comment (avekens) [inline set.mm:52449]: Tag $( $j usage 'iunopab' avoids 'ax-sep' 'ax-nul' 'ax-pr'; $) missing.

File: set.mm (hunk 1 above)
Comment (avekens) [reply set.mm:52468]: I think "Proof shortened" is sufficient.

File: set.mm [hunk 3]
```diff
@@ -54342,11 +54408,20 @@ of the restricted converse (see ~ cnvrescnv ).  (Contributed by NM,
...
-    $d A x y $.
+    $d x y $.
     $( Membership in an ordered-pair class abstraction implies membership in a
        Cartesian product.  (Contributed by Alexander van der Vekens,
-       23-Jun-2018.) $)
+       23-Jun-2018.)  Avoid ~ ax-sep , ~ ax-nul , ~ ax-pr .  (Revised by SN,
+       11-Dec-2024.) $)
     elopaelxp $p |- ( A e. { <. x , y >. | ps } -> A e. ( _V X. _V ) ) $=
+      ( copab cvv cxp cv wcel wa vex pm3.2i a1i ssopab2i df-xp sseqtrri sseli )
+      ABCEZFFGZDRBHFIZCHFIZJZBCESAUBBCUBATUABKCKLMNBCFFOPQ $.
```
Comment (avekens) [inline set.mm:54418]: Tag $( $j usage 'elopaelxp' avoids ... $) missing.

File: set.mm (hunk 1 above)
Comment (icecream17) [reply set.mm:52468]: To be honest I was debating not making a tag/old version at all since the proof is automatically found with `improve all /3`

**Review by avekens — APPROVED** [review]

File: set.mm (hunk 1 above)
Comment (wlammen) [reply set.mm:52468]: Thanks for being honest in this question.  As a possible solution here we have tags like (Proof shortened by OpenAI, 25-Mar-2020.).

**Pull request merged**